inference-engine/ie_bridges/python/src/openvino/inference_engine/CMakeLists.txt

```diff
@@ -7,18 +7,33 @@ set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${PYTHON_BRIDGE_OUTPUT_DIRECTORY}/inference_e
 
 file(GLOB SOURCE
         ${CMAKE_CURRENT_SOURCE_DIR}/ie_api.pyx
-        ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
+        ${CMAKE_CURRENT_SOURCE_DIR}/ie_api_impl.cpp)
+
+file(GLOB OFFLINE_SOURCE
+        ${CMAKE_CURRENT_SOURCE_DIR}/offline_api.pyx
+        ${CMAKE_CURRENT_SOURCE_DIR}/offline_api_impl.cpp)
```

Comment:
can we separate offline api on folder level?

Reply:
I'll try

Reply:
Done.